abdm_python_integrator/abha/utils/response_util.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.response import Response
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_500_INTERNAL_SERVER_ERROR
+
+success_response_keys = ["txnId", "healthIdNumber", "auth_methods", "token",
+                         "health_card", "health_id", "authMethods"]
+
+
```

Comment:
💯  I like that you separated the utils into separate files. To reduce the verbosity, you can remove the util suffix for the files inside the utils directory.

Reply:
Same for the files inside the views dir as well.

Reply:
Addressed in bf6e106c5de62bccc0c6189d439ea98a47277cef and 4adca48230d24e8aaa59d60103876298a8c93c61